client/signup/jetpack-connect/plans.jsx

```diff
@@ -163,9 +163,9 @@ const Plans = React.createClass( {
 } );
 
 export default connect(
-	( state, props ) => {
+	state => {
 		const user = getCurrentUser( state );
-		const selectedSite = props.sites.getSelectedSite();
+		const selectedSite = getSelectedSite( state );
 
 		const searchPlanBySlug = ( planSlug ) => {
 			return getPlanBySlug( state, planSlug );
```

Comment:
Unless we need some custom dispatch logic, mapDispatchToProps can be shortened to:
https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options
```javascript
{
    goBackToWpAdmin,
    recordTracksEvent
}
```